packages/core/src/plugins/default/markbind-plugin-plantuml.ts

```diff
@@ -89,12 +92,12 @@ module.exports = {
     graphvizCheckCompleted = false;
   },
 
-  processNode: (pluginContext, node, config) => {
-    if (node.name !== 'puml') {
+  processNode: (_pluginContext: PluginContext, node: MbNode, config: NodeProcessorConfig) => {
+    if (!node.name || node.name !== 'puml') {
```

Comment:
Is this check needed since `node.name` will not be undefined in `MbNode`?

Reply:
Thanks for the catch! Removed the check